app/views/HomeCNView.js

```diff
@@ -53,6 +54,12 @@ module.exports = (HomeCNView = (function () {
       this.courses = new Courses()
       this.supermodel.trackRequest(this.courses.fetchReleased())
 
+      window.localStorage.setItem('lastUpdatedCocoStarPage', +new Date('2021-3-30 18:00:00'))
```

Comment:
what with these dates of 2021?

Reply:
this code is changed in 2021 in china branch. i don't know why our china frontend merge pr doesn't get this 😂  so I add in this pr